ios/brave-ios/App/l10n/tools/xliff-cleanup.py

```diff
@@ -1,13 +1,19 @@
 #!/usr/bin/env python3
 
+# Copyright (c) 2018 The Brave Authors. All rights reserved.
```

Comment:
2024

Reply:
This file was created in 2018 according to git, so copyright is 2018